src/tailwind.preset.js

```diff
@@ -70,6 +70,13 @@ export default /** @type {Partial<import('tailwindcss').Config>} */ ({
           inverted: '#f2f2f2',
         },
       },
+      fontSize: {
+        // Set font size to the maximum of 16px and the inherited size.
+        //
+        // On iOS, the input font size must be at least 16px to prevent the
+        // browser from zooming into it on touch.
+        'at-least-16px': 'max(16px, 100%)',
```

Comment:
Perhaps we could use a name which does not include the `16px` value, as that's a detail that we may want/have to change in future.

Perhaps something that makes it clear that there won't be zoom in touch devices, something in the lines of `prevent-zoom`, so that we can use it as `touch:text-prevent-zoom`.

Reply:
I considered something like this but `at-least-16px` makes it more obvious at a glance what the visual effect is. The 16px value has been shipped in WebKit for years so I think is unlikely to change. They can't make it larger without breaking a lot of sites and if they made it smaller, it would be a long time before we could make a change.

Reply:
We do have a `touch-minimum` size that we use, and this is analogous, except that here we are saying "at least the minimum touch size". I suppose `at-least-min-input-size` would work, albeit a bit verbose.

Reply:
Ok, thanks for the clarification. I think you are right.